saleor/csv/utils/export.py

```diff
@@ -53,8 +54,9 @@ def export_products(
 
 
 def get_filename(model_name: str, file_type: str) -> str:
-    return "{}_data_{}.{}".format(
-        model_name, timezone.now().strftime("%d_%m_%Y"), file_type
+    hash = secrets.token_hex(nbytes=3)
```

Comment:
Are we sure that this will not return any forbidden characters for the path file?

Reply:
Yes, https://docs.python.org/3/library/secrets.html#secrets.token_hex

Reply:
Nevermind, I already answered for this by myself 😄